client/signup/steps/design-type/index.jsx

```diff
@@ -37,64 +38,40 @@ class DesignTypeStep extends Component {
 
 	getChoices() {
 		const { translate } = this.props;
-		const modified = abtest( 'siteCreationStepOne' ) === 'modified';
-
-		if ( modified ) {
-			// Note: Don't make this translatable because it's only visible to English-language users.
-			return [
-				{
-					type: 'blog',
-					label: 'Start with a blog',
-					description: 'To share your ideas, stories, and photographs with your followers.',
-					image: <BlogImage />,
-				},
-				{
-					type: 'page',
-					label: 'Start with a website',
-					description: 'To promote your business organization, or brand and connect with your audience.',
-					image: <PageImage />,
-				},
-				{
-					type: 'grid',
-					label: 'Start with a portfolio',
-					description: 'To present your creative projects in a visual showcase.',
-					image: <GridImage />,
-				},
-			];
-		}
 
 		return [
-			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
-			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
-			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
+			{
+				type: 'blog',
+				label: translate( 'Start with a blog' ),
+				description: translate( 'To share your ideas, stories, and photographs with your followers.' ),
+				image: <BlogImage />,
+			},
+			{
+				type: 'page',
+				label: translate( 'Start with a website' ),
+				description: translate( 'To promote your business organization, or brand and connect with your audience.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'To promote your business, organization, or brand and connect with your audience.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).